azure_functions_worker/dispatcher.py

```diff
@@ -408,6 +409,11 @@ def _serialize_exception(self, exc):
                 logger.info('Unable to reload azure.functions. '
                             'Using default. Exception:\n{}'.format(ex))
 
+            # Change function app directory
+            if getattr(func_env_reload_request, 'function_app_directory'):
```

Comment:
Isn't `getattr(func_env_reload_request, 'function_app_directory')` equivalent to `func_env_reload_request.function_app_directory` (including the fact that both will raise the same `AttributeError` if `function_app_directory` is not provided)? If the intent here to gracefully handle the absence of `function_app_directory`, you probably need to provide a default value. Otherwise, why not just use the simpler code: `func_env_reload_request.function_app_directory`?

Reply:
Yea, good catch, I will add a default value **None** when function_app_directory is absence.